Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "heroku-ruby-buildpack"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+libcnb = "0.6.0"
+tempfile = "3.3.0"
+flate2 = "1"
+ureq = "2.2.1"
+sha2 = "0.10"
+tar = "0.4"
+toml = "0.5"
+serde = "1.0.133"
+url = "2.2.2"
+thiserror = "1.0.30"
+regex = "1.5.4"
+
+[dev-dependencies]
+libcnb-test = { git = "https://github.com/malax/libcnb" }
+ureq = "2.4.0"
```

Comment:
`ureq` is already specified in `dependencies` so doesn't need to also be in `dev-dependencies`

```suggestion
```